helm/kiam-app/templates/agent-daemonset.yaml

```diff
@@ -66,9 +66,12 @@ spec:
               name: xtables
           livenessProbe:
             httpGet:
-              path: /ping
+              path: /health?deep=1
```

Comment:
this is probabbly the most important as the `/ping` only checks is agent is running but `/health?deep=1` checks if it can successfully connect to the server as well:
https://github.com/uswitch/kiam/blob/master/helm/kiam/values.yaml#L32-L35

Reply:
Can we put a comment in the yaml file explaining this and referring to the actual code? I think this would be very beneficial in the future.

Reply:
sure